### ISSUE TITLE
Reactors tab- Core List card- Control all Cores- Turn off all cores

### ISSUE THREAD

**Issue by stdye** [geoneutrinos/reactors#98]
Turn off all cores disables the contributions from existing custom cores and prevents adding the contributions of custom cores to the spec stats and spec plot

I am unable to display the spectrum and rate of custom cores without IAEA cores

**Comment by DocOtak** [comment]
Did some digging into this one....

* "Turn off all cores" is doing what I expect, setting the LF of all the cores in the internal core list to 0. This includes any custom cores added before the "Turn off all cores" button is pressed.
* Turning off all the cores does not _remove_ any cores from the list so you are able to find and turn back on individual cores, including custom ones.
* The "add custom" core button will not overwrite existing cores of the same name, so you cannot e.g. add a second "ATR" of the exact same name. This is the an area I think we can improve this by letting you know that some name already exists on the list.

Also keep in mind that the core list by default sorts by distance from the detector, the two research reactor presets are very far away (6000km+) from the default detector site of Boulby.

**Comment by stdye** [comment]
Order of operations seems critical. I got stuck trying to display only the spectrum and rate for a custom core because I added the custom core before hitting the "Turn off all cores" button.

1) The Core Direction Plot with 0 (1) core on displays cos(theta) values from 0 to 6 (2) but cos(theta)>1 is not possible. The cos(theta) range on the plot needs to be fixed to only display values [-1,1].

2) Listing "Use IAEA LF data for all cores" in the drop down menu for "Control all Cores" led me to think that the other two options "Use 100% Load for all cores" and "Turn off all cores" applied only to the IAEA cores. I now see it is coded differently. One way to improve access to "Custom Cores" would be to more fully isolate them from the IAEA cores. How about we make the green "Control all Cores" button work only for IAEA cores? So the green button controls "IAEA Cores" and the blue button controls "Custom Cores."
